src/test/java/com/github/fred84/queue/RedisDelayedEventServiceTest.java

```diff
@@ -239,6 +243,26 @@ void backPressureIsApplied() throws InterruptedException {
         assertThat(connection.zcard(DELAYED_QUEUE), greaterThan(5L));
     }
 
+    @Test
+    void verifyLogContext() throws InterruptedException {
```

Comment:
Is log context transient or same for all messages (all handlers) ?
What if several handlers change context simultaneously from different threads, futures ? 
Tests are required.